certified-connectors/VirusTotal/apiDefinition.swagger.json

```diff
@@ -0,0 +1,3443 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.4",
+    "title": "Virus Total APIV3",
+    "description": "user will be able to enter an ip address and get the exact experience of a report in virus total on this IP.",
```

Comment:
- Start connector description with upper case letter.
- User upper case for acronyms like `IP`.
- Replace `virus total` with `VirusTotal`.
- What does `on this IP.` refer to?

Reply:
Done